web/src/app/pages/create-survey/create-survey.component.ts

```diff
@@ -218,18 +241,44 @@ export class CreateSurveyComponent implements OnInit {
   @ViewChild('jobDetails')
   jobDetails?: JobDetailsComponent;
 
+  @ViewChild('loiPermissions')
+  loiPermissions?: LoiPermissionsComponent;
+
+  private getExistingJob(): Job {
```

Comment:
The current designs allow multiple jobs to be added through the creation flow, so we should have a reference to the current job. @DaoyuT is this supported by the current impl?

Reply:
We may not have implemented it yet, but on the last page of the creation flow you should be able to create another job:

![image](https://github.com/google/ground-platform/assets/228050/0efc3d30-20b2-4bdf-97e2-64c6c8197698)

If this isn't implemented yet, could you please add a `TODO(#bugno)`? To make the role of the function a little clearer we could also call it `getFirstJob()`.